Gopkg.lock

```diff
@@ -1,6 +1,12 @@
 # This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
```

Comment:
Just wondering why so many changes to vendor? Any idea which package has led to this changes?

Reply:
`_ "k8s.io/client-go/plugin/pkg/client/auth/gcp" ` for gcp, we have to add this plugin.
It depends on many packages.